### ISSUE TITLE
Requires `requests`

### ISSUE THREAD

**Issue by pylang** [RPiAwesomeness/BeamBot#19]
I found I needed to `pip install requests`.  You may want to add this to the README under "Required Modules".  Works great.  I'm really liking "PyBot" so far.

On another note, have you considered adding these requirements to `setup.py` under the `requires` key?

**Comment by RPiAwesomeness** [comment]
You're right, that is needed. I'll definitely consider this for a later release.

As for the requirements in `setup.py`, that's not the type of python `setup.py` script. It's a custom one I wrote for setting up the bot. However, I'll definitely look into adding a section that handles module installation!

**Comment by pylang** [comment]
Thanks.  I see now you have included `requests` in your beambot-rewrite.  

I see `setup.py` interacts with the command prompt.  Perhaps a simpler alternative to the `install_requires` key in your custom `setup.py` might be listing dependencies in a separate requirements.txt file, i.e.

```python
# requirements.txt

websockets
requests
```

Then the user can easily install any listed dependencies with a single command:

`pip install -r requirements.txt`

**Comment by RPiAwesomeness** [comment]
Good idea! Definitely will do these in the future.
